app/src/main/java/com/kabouzeid/gramophone/ui/activities/AlbumDetailActivity.java

```diff
@@ -268,11 +268,11 @@ public void onResponse(Call<LastFmAlbum> call, Response<LastFmAlbum> response) {
                         wikiReady();
                         return;
                     }
-                    else if(call.request().url().queryParameter("lang") != null) {
-                        //If the "lang" parameter is set and no wiki is given, retry with default language
-                        lastFMRestClient.getApiService().getAlbumInfo(getAlbum().getTitle(), getAlbum().getArtistName(), null).enqueue(this);
-                        return;
-                    }
+                }
+                if(call.request().url().queryParameter("lang") != null) {
```

Comment:
No because there's a return if the biography is found. Otherwise it will check if `lang` is set, and if yes it will retry without `lang`.